src/OrchardCore/OrchardCore.Data.YesSql/OrchardCoreBuilderExtensions.cs

```diff
@@ -151,21 +151,24 @@ public static OrchardCoreBuilder AddDataAccess(this OrchardCoreBuilder builder)
 
                 services.AddScoped<IDocumentStore, DocumentStore>();
                 services.AddSingleton<IFileDocumentStore, FileDocumentStore>();
-
                 services.AddTransient<IDbConnectionAccessor, DbConnectionAccessor>();
             });
 
             return builder;
         }
 
-        private static IConfiguration GetStoreConfiguration(IServiceProvider sp, YesSqlOptions yesSqlOptions)
+        private static IConfiguration GetStoreConfiguration(IServiceProvider sp, YesSqlOptions yesSqlOptions, DatabaseTableOptions databaseTableOptions)
         {
+            var tableNameFactory = sp.GetRequiredService<ITableNameConventionFactory>();
+
             var storeConfiguration = new YesSql.Configuration
             {
                 CommandsPageSize = yesSqlOptions.CommandsPageSize,
                 QueryGatingEnabled = yesSqlOptions.QueryGatingEnabled,
                 ContentSerializer = new PoolingJsonContentSerializer(sp.GetService<ArrayPool<char>>()),
-                TableNameConvention = yesSqlOptions.TableNameConvention
+                TableNameConvention = tableNameFactory.Create(databaseTableOptions),
+                Schema = databaseTableOptions.Schema,
+                IdentityColumnSize = databaseTableOptions.IdentityColumnType,
```

Comment:
> I like the comma at the end of the last initializer line, some analyzers suggest it ;)

Still I don't what's the value of it, unless it make it easier when we add a new initialization, super useful for lazy people ;)

Reply:
@hishamco adding comma at the end will prevent you from seeing red line when you are looking at the diff since you never have to touch the previous line. I also like using the training comma when possible